manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: bitvise.SSHClient
+PackageVersion: 8.49
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silentWithProgress
+Protocols:
+- ssh
+FileExtensions:
+- tlp
+- bscp
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://s3.amazonaws.com/dl.bitvise.com/BvSshClient-Inst.exe
```

Comment:
```suggestion
  InstallerUrl: https://dl.bitvise.com/BvSshClient-Inst.exe
```

Reply:
Why should I change that? The distribution via AWS is surely more stable.

Reply:
We have policies regarding the domain an installer comes from. Automated checks will trigger. It's also possible the URL suggested by @vedantmgoyal2009 is a redirect, so the binary location may move, and the s3 bucked might be dropped while the dl.bitvise.com URL stays consistent and redirects to the new location.